scripts/serve-laravel.sh

```diff
@@ -2,15 +2,17 @@
 
 mkdir /etc/nginx/ssl 2>/dev/null
 
+safe_1=${1//\ /_}
```

Comment:
If we're naming this can we use the opportunity to give it a more descriptive name?